src/status_im/data_store/realm/schemas/account/v16/core.cljs

```diff
@@ -64,7 +64,10 @@
   (log/debug "migrating v16 account database: " old-realm new-realm)
   (when-let [wallet-chat (chat-by-id new-realm "wallet")]
     (.delete new-realm wallet-chat))
-  (when-let [wallet-contact (chat-by-id new-realm "wallet")]
+  (when-let [wallet-contact (some-> new-realm
+                                    (.objects "contact")
```

Comment:
Later it would be great to have nicer functions encapsulating those `realm` tricks